Frameworks/UIKit/UIScrollView.mm

```diff
@@ -250,6 +291,14 @@ - (void)_initScrollViewer:(WXFrameworkElement*)xamlElement {
     _scrollViewer.maxZoomFactor = 1.0f;
     _scrollViewer.minZoomFactor = 1.0f;
     _scrollViewer.zoomMode = WXCZoomModeDisabled;
+
+    // setting transparent background so we can receive touch input
+    WUColor* convertedColor = ConvertUIColorToWUColor([UIColor clearColor]);
+    WUXMSolidColorBrush* brush = [WUXMSolidColorBrush makeInstanceWithColor:convertedColor];
```

Comment:
how about just setting directly to the projected transparent brush rather than creating a UIColor only to convert to the projected type?  Something like [WUXMSolidColorBrush makeInstanceWithColor:[WUColor transparent]] ?